samples/AspNetIntegration/MultipleOutputBindings.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace AspNetIntegration
+{
+    //<docsnippet_aspnetcore_multiple_outputs>
+    public class MultipleOutputBindings
+    {
+        private readonly ILogger<MultipleOutputBindings> _logger;
+        public MultipleOutputBindings(ILogger<MultipleOutputBindings> logger)
+        {
+            _logger = logger;
+        }
+        [Function("MultipleOutputBindings")]
+        public MyOutputType Run([HttpTrigger(AuthorizationLevel.Function, "post")] HttpRequest req)
+        {
+            _logger.LogInformation("C# HTTP trigger function processed a request.");
+            var myObject = new MyOutputType
+            {
+                Result = new OkObjectResult("C# HTTP trigger function processed a request."),
+                MessageText = "some output"
+            };
+            return myObject;
```

Comment:
nit q: should we inline the `myObject` variable so that we can reduce 1 line? Thinking that because this code snippet goes in the docs and I think it is good to have less number of lines.